test/classes/initializers/bothInitCalls.future

```diff
@@ -1,5 +0,0 @@
-feature request: new initializer story
-
```

Comment:
For sure!  They're mostly like this because I anticipate their error mode changing each time I make a fix and I didn't want to have to maintain them for that granularity